README.md

```diff
@@ -1,6 +1,19 @@
-# worldcubeassociation.org [![Build Status](https://github.com/thewca/worldcubeassociation.org/actions/workflows/ruby.yml/badge.svg?event=push)](https://github.com/thewca/worldcubeassociation.org/actions/workflows/ruby.yml) [![Coverage Status](https://coveralls.io/repos/github/thewca/worldcubeassociation.org/badge.svg?branch=master)](https://coveralls.io/github/thewca/worldcubeassociation.org?branch=master)
+<h1 align="center">WorldCubeAssociation Website</h1>
+<div align="center">
+  <img height="120" src="client/public/favicon.png" />
+</div>
+<br />
+<div align="center">
+  <strong>
+    This repository contains all of the code that runs on [worldcubeassociation.org](https://www.worldcubeassociation.org/).
+  </strong>
+</div>
+<br />
+<div align="center">
```

Comment:
You're opening a `div` here but you never close it

Reply:
Div now closes